### DISCUSSION TITLE
Heads up: Tilt lazy loading of tilt/handlebars is being deprecated

### DISCUSSION THREAD

**Discussion by jeremyevans** [gi/tilt-handlebars#29] [General]
I wanted to let you know that I've deprecated the lazy loading of tilt/handlebars by default in tilt.  See https://github.com/jeremyevans/tilt/commit/98a75d31e0f1cdde72b917cf60b02080e16d5427.  To allow for a transition period, the deprecated support will remain until tilt 3 is released (no ETA yet, and unlikely to be this year).  I recommend that you adjust your documentation to explain to users that tilt/handlebars should be required manually before using the tilt integration.

**Comment by gi** [comment]
@jeremyevans Thank you for the heads up.
